Interop/Runtime/src/native/kotlin/kotlinx/cinterop/NativeTypes.kt

```diff
@@ -56,6 +56,20 @@ external fun CPointer<*>.getRawValue(): NativePtr
 
 internal fun CPointer<*>.cPointerToString() = "CPointer(raw=$rawValue)"
 
+
+public final class NativeVector private constructor()
+
+public class VectorVarOf<T : NativeVector>(rawPtr: NativePtr) : CVariable(rawPtr)
```

Comment:
Generally names of `*Var(Of)` types should be prefixed with names of corresponding value classes, e.g. `NativeVectorVarOf`.